gradle/configureSourceSets.gradle

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+kotlin {
+    targets {
+        fromPreset(presets.jvm, 'jvm') {
+            withJava()
+            configure([compilations.main, compilations.test]) {
+                kotlinOptions {
+                    jvmTarget = '1.6'
+                }
+            }
+        }
+        fromPreset(presets.js, 'js') {
+            nodejs {}
+            configure([compilations.main, compilations.test]) {
+                kotlinOptions {
+                    sourceMap = true
+                    moduleKind = "umd"
+                    metaInfo = true
+                }
+            }
+        }
+    }
+
+    sourceSets.all {
+        kotlin.srcDirs = ["$it.name/src"]
+        resources.srcDirs = ["$it.name/resources"]
+        languageSettings {
+            progressiveMode = true
+            useExperimentalAnnotation("kotlin.Experimental")
+            useExperimentalAnnotation("kotlin.ExperimentalMultiplatform")
+            useExperimentalAnnotation("kotlin.ExperimentalStdlibApi")
+            useExperimentalAnnotation("kotlinx.serialization.InternalSerializationApi")
+        }
+    }
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                api 'org.jetbrains.kotlin:kotlin-stdlib-common'
+//                api project(":kotlinx-serialization-runtime")
```

Comment:
Better remove this line at all or comment about it (smth like 'runtime can't depend on itself' or whatever)